### DISCUSSION TITLE
Twitch integration ?

### DISCUSSION THREAD

**Discussion by Rengrouze** [FakeFishGames/Barotrauma#7048] [Ideas and suggestions]
is this something you think about for "hopes and dreams" ?

I think the game is very suitable for this kind of stuff

**Comment by James-E-A** [comment]
What kind of integrations are you hoping for? Anything in particular that isn't possible with the existing modding interface?

**Reply by Rengrouze** [reply]
Well are you familiar with the twitch integration of vermintide 2 ? it allows viewers to choose between 2 events, one good or bad, or both good, both bad,( like "Hey Viewers vote, 1- The group is attacked by a giant Rat OR 2- The group is attacked by a massive horde") i think this can be really fun to implement on the game, don't know if it's possible by modding btw

**Comment by Keltskiy** [comment]
Could see this being fun for content creation, events like being attacked by monsters or contracting a disiease or changing the streamers role to a traitor or something
